packages/webpack-config-single-spa-react/lib/webpack-config-single-spa-react.js

```diff
@@ -0,0 +1,104 @@
+const path = require("path");
+const CleanWebpackPlugin = require("clean-webpack-plugin").CleanWebpackPlugin;
+const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
+const { UnusedFilesWebpackPlugin } = require('unused-files-webpack-plugin')
+
+module.exports = webpackConfigSingleSpaReact;
+
+function webpackConfigSingleSpaReact(opts) {
```

Comment:
Any desire to use webpack merge or something like that so people can extend it if they want?

Reply:
We are using webpack-merge - just inside of the user's webpack.config.js file instead of inside of webpack-config-single-spa-react. See

Reply:
ah gotcha